bobtask/map.go

```diff
@@ -134,3 +134,34 @@ func (tm Map) CollectNixDependenciesForTasks(whitelist []string) ([]nix.Dependen
 
 	return nixDependencies, nil
 }
+
+// ChildrenForTask gets all children for task by name
+func (tm Map) ChildrenForTask(name string) Map {
+	children := make(Map, 0)
+
+	collectChildren(name, tm, children)
+
+	return children
+}
+
+// collectChildren collects all children for a task by name inside children container
+// from allTasks
+func collectChildren(name string, allTasks Map, children Map) {
```

Comment:
Isn't it possible to reuse the walk function for tasks? It already iterates through al child tasks https://github.com/benchkram/bob/blob/5127d793ae58a7c27701ef7f2ef2c136a0bb07fc/bobtask/map.go#L21